src/Exception/ShouldNotHappenException.php

```diff
@@ -33,7 +33,6 @@ private function createDefaultMessageWithLocation(): string
         $method = $class !== null ? ($class . '::' . $function) : $function;
 
         /** @var string $method */
-        /** @var int $line */
```

Comment:
mark previous docs as comment seems make this `@var int $line` removed.

Reply:
Resolved with apply setDocComment after set comments attributes https://github.com/rectorphp/rector-src/pull/3263/commits/2413173a664c15e79d5c99d36105ead0d4c12eb3